addons/website/static/src/scss/website.scss

```
@@ -406,6 +406,12 @@ $-header-nav-link-height: $nav-link-height;
     }
 }
 
+hr {
+    border-top: $hr-height solid $hr-color;
+    color: inherit;
```

This deserves a comment. Somethink like
```suggestion
    // Move the set up color as a border color. Indeed, we may use color-mix to
    // define that color as a function of currentColor, in which case Safari is
    // known to crash when using that to set the color property.
    border-top: $hr-height solid $hr-color;
    color: inherit;
```

---

Done, but is it really necessary since it's explained in the commit message?

---

Yes, the code should be understandable on its own. Without GitHub history. Having to look the git history to understand the code is a symptom of bad cod or badly documented code. Caring about having a clean git history is important but IMO a 0.1% increase in code quality justify any break in git history (but I know that many don't agree with that at Odoo).

Anyways, in this case: adding the comment has only advantages as it increases code quality without hindering the git history.